### PR TITLE
Add make target to render keda-manager manifest

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- add make target to render keda manager manifest yaml file. Such target is quite handy in generating `keda-manager.yaml` release artifact

 see [here](https://github.com/kyma-project/keda-manager/releases/tag/v0.0.1)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -113,6 +113,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+.PHONY: render-manifest
+render-manifest: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```
Comment (anoipm) [inline Makefile:117]: Very similar target in kyma/components/function-controller is named "manifests". Is this new way of naming or could we standarize these names?

Comment (anoipm) [inline Makefile:117]: Help text with "Deploy" said me that this yaml will be deployed. IMO this target only create yaml without deploying. Could we change this description?

Comment (kwiatekus) [reply Makefile]: fixed

Comment (anoipm) [reply Makefile]: ok

**Review by anoipm — APPROVED** [review]

**Pull request merged**